svid/jwtsvid/tests/check_svid.c

```diff
@@ -310,6 +236,62 @@ START_TEST(test_jwtsvid_error5)
 }
 END_TEST
 
+// precondition: invalid jwt token with subject not spifeeid
+START_TEST(test_jwtsvid_error_subject_not_spifeeid)
+{
+    char token[]
+        = "eyJhbGciOiJFUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6ImZmM2M1Yzk2LTM5MmUtNDZ"
+          "lZi1hODM5LTZmZjE2MDI3YWY3OCJ9."
+          "eyJzdWIiOiJodHRwOi8vZXhhbXBsZS5jb20vd29ya2xvYWQxIiwibmFtZSI6IkpvaG4g"
+          "RG9lIiwiaWF0IjoxNTE2MjM5MDIyLCJleHAiOjE2MjAwMDAwMDB9.cVivPwGf8M9AkHP"
+          "9ekqeeWeDpeVHMV6A4TrdbHx285Ga07C4o47UHPqQvuttQACKnT7h_uhujJEFGUph0oJwqg";
+
+    err_t err;
+    jwtsvid_SVID *svid = jwtsvid_ParseInsecure(token, NULL, &err);
+    ck_assert_ptr_eq(svid, NULL);
+    jwtsvid_SVID_Free(svid);
+
+}
+END_TEST
+
+// precondition: invalid jwt token without expiration date
+START_TEST(test_jwtsvid_error_without_exp)
+{
+    char token[]
+        = "eyJhbGciOiJFUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6ImZmM2M1Yzk2LTM5MmUtNDZlZ"
+          "i1hODM5LTZmZjE2MDI3YWY3OCJ9."
+          "eyJzdWIiOiJzcGlmZmU6Ly9leGFtcGxlLmNvbS93b3JrbG9hZDEiLCJuYW1lIjoiSm9ob"
+          "iBEb2UiLCJpYXQiOjE1MTYyMzkwMjJ9."
+          "tIyeP2Alj6OE1KN3sLGDA89EdkTPqoHMi1hMyARyJZ_wpOpL57dgeeMSXWjbHYLW6tKIT"
+          "iOA9pcQVrCQRbcY9A";
+
+    err_t err;
+    jwtsvid_SVID *svid = jwtsvid_ParseInsecure(token, NULL, &err);
+    ck_assert_ptr_eq(svid, NULL);
+    jwtsvid_SVID_Free(svid);
+
+}
+END_TEST
+
+// precondition: invalid jwt token without issuer, jti, audience
+START_TEST(test_jwtsvid_error_issuer_jti_aud)
+{
+    char token[]
+        = "eyJhbGciOiJFUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6ImZmM2M1Yzk2LTM5MmUtNDZlZ"
+        "i1hODM5LTZmZjE2MDI3YWY3OCJ9."
+        "eyJzdWIiOiJzcGlmZmU6Ly9leGFtcGxlLmNvbS93b3JrbG9hZDEiLCJuYW1lIjoiSm9obiB"
+        "Eb2UiLCJpc3MiOiJIUEUiLCJpYXQiOjE1MTYyMzkwMjIsImp0aSI6IkFCQ0RFRkdIIiwibmJ"
+        "mIjoxNDAwMDAwMDAwLCJhdWQiOlsiYXVkMSIsImF1ZDIiXX0.afcc79kL3JYqwrh1ABQe7og"
+        "Cy-WxWGb-G2YdmIKIeej3RMT-usawYCC8c1OH30Fi-_ygfzk_qLJnOs1t30tcEQ";
+
+    err_t err;
+    jwtsvid_SVID *svid = jwtsvid_ParseInsecure(token, NULL, &err);
```

Comment:
Check for error

Reply:
Added checked error, we will define documentation for errors.